mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -246,10 +246,10 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		ARG 1 yaw
 	METHOD method_5637 isWet ()Z
 		COMMENT Returns whether this entity is touching water, or is being rained on, or is inside a bubble column...
-		COMMENT 
-		COMMENT @see net.minecraft.entity.Entity#isTouchingWater() 
-		COMMENT @see net.minecraft.entity.Entity#isBeingRainedOn() 
-		COMMENT @see net.minecraft.entity.Entity#isInsideBubbleColumn() 
+		COMMENT
+		COMMENT @see net.minecraft.entity.Entity
+		COMMENT @see net.minecraft.entity.Entity
+		COMMENT @see net.minecraft.entity.Entity
```

Comment:
Yep Engima tripped balls again

Reply:
Maybe Enigma really wants you to see `Entity`?